bin/utils.py

```diff
@@ -4,21 +4,28 @@
     ('cs', 'csharp'),
     ('css', 'css'),
     ('diff', 'diff'),
+    ('erl', 'erlang'),
+    ('ex', 'elixir'),
     ('go', 'go'),
     ('h', 'objectivec'),
+    ('hs', 'haskel'),
```

Comment:
Haskell is spelled with 2 L
Here it has only one, is it how it's spelled in pygments?

Reply:
`haskell` in pygments.